packages/flow-core/src/components/f-grid/f-grid.test.ts

```diff
@@ -0,0 +1,16 @@
+import { html, fixture, expect } from "@open-wc/testing";
+// importing flow-core components
+import "@cldcvr/flow-core";
+
+import { FGrid } from "@cldcvr/flow-core";
+
+describe("f-grid", () => {
+	it("is defined", () => {
+		const el = document.createElement("f-grid");
+		expect(el).instanceOf(FGrid);
+	});
+	it("should render with default values", async () => {
+		const el = await fixture(html` <f-grid></f-grid> `);
+		expect(el.getAttribute("gap")).to.equal("small");
```

Comment:
write a test to check the inner item's offsetWidth based on `minCellWidth` and `maxCellWidth `.

same for `cellHeight `

Reply:
getting inconsistent widths in `@open-wc/testing`, so skipping this part